scripts/publish_vessel_info

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Add the `set -e`, we are validating if every step fails or not, just in case in the future we add some line and we forget to add the validation.